Update audit-grafana.md (#72297)
grafana/grafana: docs/sources/setup-grafana/configure-security/audit-grafana.md

### Console exporter

> Audit logs are sent to the Grafana default logger. The audit logs use the `auditing.console` logger and are logged on `debug`-level, learn how to enable debug logging in the [log configuration]({{< relref "../configure-grafana#log" >}}) section of the documentation. Accessing the audit logs in this way is not recommended for production use. For production use you may want to consider using `filters` as follows.

Reviewer: ```suggestion
The audit logs are sent to the Grafana default logger using the `auditing.console` logger and are logged at the `debug` level. To enable debug logging, refer to [log configuration]({{< relref "../configure-grafana#log" >}}). 

Accessing the audit logs in this way is not recommended for production use. For production use, consider using `filters`.

For example:
```
